### PR TITLE
!fix: Auth0 now allow us to get the access_token

### PR DESCRIPTION
**Pull request by ChristianECG**

### CODE REVIEW

File: components/security/header/Header.jsx [hunk 1]
```diff
@@ -1,15 +1,48 @@
...
 import styles from "../../../styles/Header.module.scss";
 import position from "../../../styles/Profile.module.scss";
 import styles2 from "../../../styles/Userdata.module.scss";
 
+let auth0 = {
+	domain: 'https://gethired.us.auth0.com',
+	response_type: 'token',
+	client_id: '9IKIwhKs44cPB0VceKTaVHZnXRSd2zUJ',
+	redirect_uri: ''
+}
```
Comment (luismardz) [inline components/security/header/Header.jsx:14]: Pull this values from an env file

File: components/security/header/Header.jsx [hunk 2]
```diff
@@ -1,15 +1,48 @@
...
+
 const Header = () => {
-	const { user } = useUser();
 
+	let [user, setUser] = React.useState(null);
+	React.useEffect(async () => {
```
Comment (luismardz) [inline components/security/header/Header.jsx:19]: We might want to set the user data as context, for everybody to be able to consume

File: components/security/header/Header.jsx [hunk 3]
```diff
@@ -1,15 +1,48 @@
...
+			const urlParams = new URLSearchParams(window.location.hash.replace('#', ''));
+			const accessToken = urlParams.get('access_token');
+			if(accessToken) {
+				localStorage.setItem('access_token', accessToken);
+				window.location = window.location.origin;
+				return;
+			}
+		}
+
+		if(localStorage.getItem('access_token') && !user) {
```
Comment (luismardz) [inline components/security/header/Header.jsx:32]: 👍

File: components/security/header/Header.jsx (hunk 1 above)
Comment (ChristianECG) [reply components/security/header/Header.jsx:14]: env data it's not available for components. I can't get it for this config, and it's not sensible data.

File: components/security/header/Header.jsx (hunk 2 above)
Comment (ChristianECG) [reply components/security/header/Header.jsx:19]: user variable is only for header info. With access_token, backend devs can identify the user and return the relevant data for every endpoint.

**Review by luismardz — APPROVED** [review]

**Review by luismardz — APPROVED** [review]

**Review by luismardz — APPROVED** [review]

**Review by elyager — APPROVED** [review]

**Pull request merged**